handlers/error.go

```diff
@@ -61,6 +61,7 @@ const (
 	ErrFetchKubernetesCode      = "2051"
 	ErrPanicRecoveryCode        = "2052"
 	ErrBlankNameCode            = "2053"
+	ErrSaveSessionCode          = "2054"
```

Comment:
@Chadha93 you should update this to some alphabetic string like `"replace"`. CI will take care of updating this to an integer now.

Reply:
@sayantan1413 @Shreyas220 quick check, we aren't approving changes with integer error codes now, right? Since https://github.com/meshery/meshery/pull/3527 is merged and running successfully.

Reply:
Yes @DelusionalOptimist I completely forgot about that so as #3527 is merged so we need to use "replace_me" from now on. So @Chadha93 please change all the error codes from integer to a placeholder string like "replace_me". It's completely my fault that I didn't inform you this @Chadha93 .

Reply:
@Shreyas220 yoy will have this in a updates contributing doc soon, is that right?

Reply:
@DelusionalOptimist, Yes it's a good approach to have it in string format. Thanks, I'll update that